aspnet/mvc/controllers/areas.rst

```diff
@@ -3,10 +3,12 @@ Areas
 
 By `Tom Archer`
 
-Areas provide a way to separate a large MVC application into semantically-related groups of models, views, and controllers. Let's take a look at an example to illustrate how Areas are created and used. Let's say you have a store app that has two distinct groupings of controllers and views: Products and Services.
+Areas provide a way to separate a large MVC application into semantically-related groups of models, views, and controllers. 
 
 Instead of having all of the controllers located under the Controllers parent directory, and all the views located under the Views parent directory, you could use Areas to group your views and controllers according to the area (or logical grouping) with which they're associated.
 
+Let's take a look at an example to illustrate how Areas are created and used. Let's say you have a store app that has two distinct groupings of controllers and views: Products and Services. A typical folder structure for that using MVC areas looks like below:
```

Comment:
> Do you really want both the products and services areas to feature Home controllers?

Why not? Our route template is generally like `{area}/{controller=Home}/{action=Index}`...every area having a home controller seems normal to me..